bapsflib/utils/__init__.py

```diff
@@ -11,10 +11,24 @@
 """
 Package of developer utilities.
 """
-__all__ = ['decorators', 'errors', 'warnings']
+__all__ = ['decorators', 'errors', 'warnings', 'NdarrayToXarray']
 
 from . import (decorators, errors, warnings)
 
 
-def foo(x):
-    return x
+def NdarrayToXarray(data, arr_name={}):
+    import xarray as xr
```

Comment:
move between `__all__` and the "from" imports

All imports should be at the top of the file, unless you're trying to avoid things like cyclical imports.